Smarty/templates/PivotView.tpl

```diff
@@ -121,6 +122,24 @@ $(function() {
 						fields = Array();
 					}
 				}
+			},
+			onRefresh: function(config) {
+				let rowFields = JSON.parse(rowlinks);
+				$('.pvtRowLabel').click(function(e) {
+					let vTotalAxisLabels = $('.pvtAxisLabel').length;
+					let vTotalRowLabels =  ($(this).siblings('th').length)+1;
+					let vColRowLabels = vTotalAxisLabels - vTotalRowLabels ;
+					let thHeaderName = $('.pvtAxisLabel').eq($(this).index()+vColRowLabels);
+					const advSearch = [{
+						'columnname': rowFields[thHeaderName.html()],
+						'comparator': 'e',
+						'value': e.target.innerText,
+						'groupid': 1,
+						'columncondition': 'and'
+					}];
+					const urlstring = `${JSON.stringify(advSearch)}&advft_criteria_groups=[null,{"groupcondition":""}]`;
```

Comment:
don't we have to use encodeuricomponent here?  For example, if `e.target.innerText` or the header has a value of `me&you` I think that ampersand will break the URL.

Reply:
Updated :D